addons/website_sale/controllers/backend.py

```diff
@@ -89,7 +89,7 @@ def fetch_dashboard_data(self, date_from, date_to):
                 ('state', '=', 'draft'),
                 ('order_line', '!=', 'False'),
                 ('partner_id', '!=', request.env.ref('base.public_partner').id),
-                ('date_order', '<=', fields.Datetime.to_string(datetime.now() - timedelta(hours=1))),
+                ('date_order', '<=', fields.Datetime.to_string(datetime.utcnow() - timedelta(hours=1))),
```

Comment:
Problem is that it returns a strings which need to be re-converted to use in timedelta calculations.